code/modules/power/heavy_emitter.dm

```diff
@@ -0,0 +1,270 @@
+/obj/machinery/power/heavy_emitter
+	name = "Heavy Emitter Base"
+	desc = "This should never fucking exist!"
+	icon = 'icons/obj/heavy_emitter.dmi'
+	icon_state = "centre_off"
+	anchored = FALSE
+	density = TRUE
+	payment_department = ACCOUNT_ENG
+
+/obj/machinery/power/heavy_emitter/wrench_act(mob/living/user, obj/item/I)
+	. = ..()
+	if(anchored)
+		return FALSE
+	I.play_tool_sound(src, 50)
+	to_chat(user, "<span class='notice'>You start to rotate [src].</span>")
+	if(do_after(user,10 SECONDS,FALSE,src))
+		setDir(turn(dir,-90))
+	return TRUE
+
+/obj/machinery/power/heavy_emitter/examine(mob/user)
+	. = ..()
+	if(anchored)
+		. += "It is welded to the floor"
+
+/obj/machinery/power/heavy_emitter/welder_act(mob/living/user, obj/item/I)
+	. = ..()
+	I.play_tool_sound(src, 50)
+	to_chat(user, "<span class='notice'>You start welding [src].</span>")
+	if(do_after(user,10 SECONDS,FALSE,src))
+		anchored = !anchored
+	return TRUE
+
+/obj/machinery/power/heavy_emitter/proc/check_part_connectivity()
+	return TRUE
+
+/obj/machinery/power/heavy_emitter/proc/turn_on()
+	return
+
+/obj/machinery/power/heavy_emitter/proc/turn_off()
+	return
+
+/obj/machinery/power/heavy_emitter/centre
+	name = "Heavy Emitter Core"
+	desc = "Dangerously unstable, military grade capacitor that eats power like it's candy and then releases an incredibly potent burst of energy that can anihiliate anything."
```

Comment:
"A dangerously unstable, military grade capacitor that eats power like it's candy before releasing an incredibly potent burst of energy that can annihilate anything."